sdk/storage/azure-storage-file-datalake/azure/storage/filedatalake/_serialize.py

```diff
@@ -18,7 +18,8 @@
     '2020-04-08',
     '2020-06-12',
     '2020-08-04',
-    '2020-10-02'
+    '2020-10-02',
+    '2021-02-12'
```

Comment:
I recommend doing add new version in the base branch instead of in each feature branch in the future, to keep the feature PRs from getting cluttered.